Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,387 @@
+
+  
+
+#### Maximum Torque at Maximum Speed: 
+
+- Using the above logic while setting magnet distance to the closest value and wheel speed to the maximum value (288 rpm), we find the maximum torque at maximum speed is 121.85 Nm. Safety always being a concern, this force is much lower than what is needed to break any leg bone in the average person. [4] [6]. 
+
+  
+
+#### Minimum Torque at Maximum Speed: 
+
+- Using the above logic while setting magnet distance to the furthest value and wheel speed to the maximum value, we find the minimum torque at maximum speed is 0.004 N•m. 
+
+#### Resistance Resolution: 
+
+- The resistance system will have a resolution of at least 85 states. 
+
+- The controller will allow for 10-bit resolution or 1024 unique states. 
+
+The linear actuator used has a length of 1.97". In practical usage, we would use 49.96 % of this range, yielding an effective number of states of 511.
+
+Note: The minimum and maximum speeds referenced within this section have been converted to bike tire rpm, so the units of measure are consistent throughout the report. The pedaling speed sourced from the reference was multiplied by the gear ratio from the front to rear sprocket, which was 2.44. The speed of the bike is referred to in the units of bike tire revolutions per minute (rpm). 
+
+
+
+
+### Buildable Schematic:
+
+![image](https://user-images.githubusercontent.com/114370750/205554450-37b7aedf-20ad-4403-ab97-4d81d6decbce.png)
+
+Figure 1. Resistance System Full View
+
+![image](https://user-images.githubusercontent.com/114370750/205554492-89368243-436b-4ada-a5b3-ee7d10c3539f.png)
+
+Figure 2. Close Up of Linear Actuator, Flywheel, Magnet Holder
+
+![image](https://user-images.githubusercontent.com/114370750/205554527-ea166ccf-6503-40b6-9993-45dad74d8431.png)
+
+Figure 3. Close Up of Linear Actuator, Flywheel, Magnet Holder with Different Viewpoint
+
+
+
+### Aluminum Flywheel:
+
+•	An aluminum circle with a radius of 101.6 mm (4”) will be cut from a sheet of aluminum material. 
+
+•	A hole will be drilled in the center of the aluminum flywheel. 
+
+•	The steel flywheel on the UNISKY bike stand will be replaced with the new aluminum flywheel. 
+
+&ensp;&thinsp; &ensp;&thinsp; •The bolt holding the steel flywheel will be unscrewed.
+   
+&ensp;&thinsp; &ensp;&thinsp; •The steel flywheel will be replaced with the aluminum flywheel.
+   
+&ensp;&thinsp; &ensp;&thinsp; •The bolt will be screwed back into place on the bike stand.
+   
+•	The drive wheel at the back of the UNISKY bike stand will be screwed into the rear wheel of the bike.
+
+&ensp;&thinsp; &ensp;&thinsp; •	This will ensure maximum transfer of energy when the magnetic breaking occurs to the flywheel. 
+   
+•	The UNISKY bike stand will come with a magnetic brake which will be removed. 
+
+
+*Bracket:*
+
+•	For the magnetic portion of the resistance system, two mounting brackets will be welded to a metal block stand.
```

Comment:
You need to use extrusion or tubing. A metal block won't work. Further, it seems you aren't considering the force that will be applied to the actuator (Newton's first law). This will need to be attached to the bike stand or it will be flung away from the bike.

However, I suggest that the mounting system for the actuator be a separate subsystem.